src/main/webapp/app/overview/result-history/result-history.component.ts

```diff
@@ -26,7 +26,7 @@ export class ResultHistoryComponent implements OnChanges {
 
     ngOnChanges(): void {
         this.showPreviousDivider = this.results.length > MAX_RESULT_HISTORY_LENGTH;
-        if (this.exercise?.type === ExerciseType.TEXT) {
+        if (this.exercise?.type === ExerciseType.TEXT || this.exercise?.type === ExerciseType.MODELING) {
```

Comment:
It might be useful to define a utility function that assigns a meaning to the condition `exercise?.type === ExerciseType.TEXT || exercise?.type === ExerciseType.MODELING` since it appears a few times in the code